src/components/NewWorkspaceModal.js

```diff
@@ -221,16 +247,16 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
           isClearable: false,
           placeholder: 'Select a billing project',
           value: namespace,
+          ariaLiveMessages: ({ onFocus: onFocusAria, onChange: onChangeAria }),
           onChange: ({ value }) => setNamespace(value),
           styles: { option: provided => ({ ...provided, padding: 10 }) },
           options: _.map(({ projectName, invalidBillingAccount, cloudPlatform }) => ({
+            'aria-label': `${getCloudContextTitle(cloudPlatform)} ${projectName}${ariaInvalidBillingAccountMsg(invalidBillingAccount)}`,
             label: h(TooltipTrigger, {
-              content: invalidBillingAccount &&
-                'Workspaces may only be created in billing projects that have a Google billing account accessible in Terra',
-              side: 'left'
+              content: invalidBillingAccount && invalidBillingAccountMsg, side: 'left'
             },
             [div({ style: { display: 'flex', alignItems: 'center' } },
-              [h(cloudContextIcon, { cloudPlatform }), projectName]
+              [h(cloudContextIcon, { cloudPlatform, key: projectName }), projectName]
```

Comment:
Setting the key to the project name (which should be unique) fixes this console warning that I just introduced in https://github.com/DataBiosphere/terra-ui/pull/3407

```
react_devtools_backend.js:4026 Warning: Each child in a list should have a unique "key" prop. See https://reactjs.org/link/warning-keys for more information.
    at div
    at TooltipTrigger (http://localhost:3000/static/js/src_appLoader_js-data_image_svg_xml_3Csvg_xmlns_27http_www_w3_org_2000_svg_27_viewBox_270_0_1-d7b3bc.chunk.js:7963:24)
    at div
    at div
    at div
```